corehq/blobs/migrate.py

```diff
@@ -240,6 +241,15 @@ def migrate(self, *args, **kw):
         hqmedia.CommCareVideo,
         hqmedia.CommCareMultimedia,
     ], CouchAttachmentMigrator),
+    Migrator("xforms", [
+        xform.XFormInstance,
+        ("XFormInstance-Deleted", xform.XFormInstance),
+        xform.XFormArchived,
+        xform.XFormDeprecated,
+        xform.XFormDuplicate,
+        xform.XFormError,
+        xform.SubmissionErrorLog,
```

Comment:
Good question. I'm not 100% sure, but I'd default to also migrating those attachments. Might be worth checking all_docs_by_doc_type to see how many of these exist?